lib/src/util/prop_errors.dart

```diff
@@ -24,10 +24,10 @@ class PropError extends Error {
   static const String combinationPrefix = 'InvalidPropCombinationError: ';
 
   final String _messagePrefix;
-  final invalidValue;
+  dynamic invalidValue;
   final String propName;
   final String prop2Name;
-  final message;
+  String message;
```

Comment:
This should stay final
```suggestion
  final String message;
```